From a4181318c403c25c2f391ef14bb03a736faa9ac8 Mon Sep 17 00:00:00 2001
From: "geoffroy.weisenhorn" <weisenhorn.geoffroy@intel.com>
Date: Tue, 6 Oct 2015 17:50:43 +0200
Subject: [PATCH] Update Netfilter configuration for M

Android Marshmallow add a new subcommand to ndc (strict), that needs
to activate following netfilter extensions connmark, owner, u32

Change-Id: I57ec6a88c7ad776c4e9a1075bbb5a0db1eab7051
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-3385
Signed-off-by: geoffroy.weisenhorn <weisenhorn.geoffroy@intel.com>
---
 arch/x86/configs/x86_64_defconfig |   12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

Index: b/arch/x86/configs/x86_64_defconfig
===================================================================
--- a/arch/x86/configs/x86_64_defconfig	2016-03-08 16:41:15.570471493 -0800
+++ b/arch/x86/configs/x86_64_defconfig	2016-03-08 16:42:51.915477349 -0800
@@ -770,7 +770,7 @@
 # CONFIG_NETFILTER_NETLINK_QUEUE is not set
 CONFIG_NETFILTER_NETLINK_LOG=y
 CONFIG_NF_CONNTRACK=y
-# CONFIG_NF_CONNTRACK_MARK is not set
+CONFIG_NF_CONNTRACK_MARK=y
 CONFIG_NF_CONNTRACK_SECMARK=y
 CONFIG_NF_CONNTRACK_PROCFS=y
 # CONFIG_NF_CONNTRACK_EVENTS is not set
@@ -806,7 +806,7 @@
 # Xtables combined modules
 #
 CONFIG_NETFILTER_XT_MARK=y
-# CONFIG_NETFILTER_XT_CONNMARK is not set
+CONFIG_NETFILTER_XT_CONNMARK=y
 
 #
 # Xtables targets
@@ -814,7 +814,7 @@
 # CONFIG_NETFILTER_XT_TARGET_AUDIT is not set
 # CONFIG_NETFILTER_XT_TARGET_CHECKSUM is not set
 # CONFIG_NETFILTER_XT_TARGET_CLASSIFY is not set
-# CONFIG_NETFILTER_XT_TARGET_CONNMARK is not set
+CONFIG_NETFILTER_XT_TARGET_CONNMARK=y
 CONFIG_NETFILTER_XT_TARGET_CONNSECMARK=y
 # CONFIG_NETFILTER_XT_TARGET_CT is not set
 # CONFIG_NETFILTER_XT_TARGET_DSCP is not set
@@ -848,7 +848,7 @@
 # CONFIG_NETFILTER_XT_MATCH_CONNBYTES is not set
 # CONFIG_NETFILTER_XT_MATCH_CONNLABEL is not set
 # CONFIG_NETFILTER_XT_MATCH_CONNLIMIT is not set
-# CONFIG_NETFILTER_XT_MATCH_CONNMARK is not set
+CONFIG_NETFILTER_XT_MATCH_CONNMARK=y
 CONFIG_NETFILTER_XT_MATCH_CONNTRACK=y
 # CONFIG_NETFILTER_XT_MATCH_CPU is not set
 # CONFIG_NETFILTER_XT_MATCH_DCCP is not set
@@ -869,7 +869,7 @@
 # CONFIG_NETFILTER_XT_MATCH_MULTIPORT is not set
 # CONFIG_NETFILTER_XT_MATCH_NFACCT is not set
 # CONFIG_NETFILTER_XT_MATCH_OSF is not set
-# CONFIG_NETFILTER_XT_MATCH_OWNER is not set
+CONFIG_NETFILTER_XT_MATCH_OWNER=y
 CONFIG_NETFILTER_XT_MATCH_POLICY=y
 # CONFIG_NETFILTER_XT_MATCH_PKTTYPE is not set
 CONFIG_NETFILTER_XT_MATCH_QTAGUID=y
@@ -885,7 +885,7 @@
 # CONFIG_NETFILTER_XT_MATCH_STRING is not set
 # CONFIG_NETFILTER_XT_MATCH_TCPMSS is not set
 # CONFIG_NETFILTER_XT_MATCH_TIME is not set
-# CONFIG_NETFILTER_XT_MATCH_U32 is not set
+CONFIG_NETFILTER_XT_MATCH_U32=y
 # CONFIG_IP_SET is not set
 # CONFIG_IP_VS is not set
 
